prov/rxm/src/rxm_attr.c

```diff
@@ -49,7 +49,7 @@
  * requested by the app. */
 
 struct fi_tx_attr rxm_tx_attr = {
-	.caps = RXM_TX_CAPS,
+	.caps = RXM_TX_CAPS | FI_HMEM,
```

Comment:
Why aren't the RXM_TX_CAPS and RXM_RX_CAPS updated instead?

Reply:
RXM_TX_CAPS is used in multiple different fi_tx_attr's, such as the collective variant, where HMEM is not supported. In addition, I just followed the collective approach which just or'ed in the addition capability.

Reply:
Why would this not work with collective support enabled?

Reply:
One thing I noticed from code inspection is util_coll doing memcpy from a user provided send and result buf. I believe this needs to be update to use HMEM memcpy variants.